Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -332,23 +333,29 @@ namespace uWS {
             memcpy(dst + headerLength, src, length);
 
             if ( !isServer ) {
-                uint32_t mask_i32 = *reinterpret_cast<uint32_t*>(mask);
+                std::byte* start_byte = dst + headerLength;
+                std::byte* end_byte   = start_byte + length;
+                size_t     i          = 0;
```

Comment:
In cases like these where a loop variable continues to be used outside of its scope it's better to pick a name that explains what it is used for.  So instead of `i` something like `start_offset` would be better.